contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -58,6 +58,21 @@ def _execute(self, module_search_string: str):
         wait_for_condition(lambda: functions_dataview.get_row_count() > 0)
 
 
+class VerifySymbolsLoaded(E2ETestCase):
+
+    def _execute(self, symbol_search_string: str):
+        logging.info('Start verifying symbols with substring %s are loaded', symbol_search_string)
+        functions_dataview = DataViewPanel(self.find_control("Group", "FunctionsDataView"))
+
+        logging.info('Filtering symbols')
+        functions_dataview.filter.set_focus()
+        functions_dataview.filter.set_edit_text('')
+        send_keys(symbol_search_string)
+        logging.info('Verifying at least one symbol with substring %s has been loaded',
+                     symbol_search_string)
+        wait_for_condition(lambda: functions_dataview.get_row_count() > 1)
```

Comment:
Is `wait_for_condition` needed here? You're waiting for the last `send_keys` to result in a filtering of the list, but the # of entries will decrease as a result of this - so if your intention is to really wait for a potential delay, this condition will probably be satisfied right from the start.

This should also do it: `self.expect_true(functions_dataview.get_row_count() > 1)`.

Reply:
No, this is actually a great suggestion: It's simpler and only what is actually needed. I was just copy & pasting stuff from other code earlier.